### DISCUSSION TITLE
闲置空间 × 游民 匹配计划

### DISCUSSION THREAD

**Discussion by Demian101** [creatorsand/.github#51] [Ideas]
有个想法有没有搞头：
706 公众号搞个推文，闲置空间 × 游民 匹配计划

- 闲置空间： 免费出让闲置住房一年（大部分在低线城市）
- 游民：免费入住，承诺一个空间大概的改造思路和成果，自主负责空间的软装和室内设计，可先探访当地二手市场，或者 stooping 🤣，低成本地去改造整个空间。想做 coliving 的，做，想做公共客厅的，做，想单纯当成游民 cozy 一站的亦可

双方撮合为主，706 提供平台，相互看对眼儿即可。

2 方可能要签个承诺合同

----

游民侧的痛点：
- 短租房极其难找
- 既需要便宜的住宿，又有软装的小想法，又不想在租住的房子上白搭太多钱。毕竟运又运不走，装又懒得装

闲置空间的痛点：
- 还没搞好软装
- 低线城市，租又租不出去；没人来
- 想搞空间，但是没经验，不会宣传，没有渠道
- ...

**Comment by ShirleneLiu** [comment]
来我家，真的有一层，可以给你搞 stooping 你做起来

**Comment by Demian101** [comment]
重庆山区吗
